.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
```

Comment:
```suggestion
- [ ] Post **Proposal (full Problem/Solution statement)** in `#expensify-open-source`
```

Reply:
If I understand correctly, in this template whenever we say "Proposal" we're referring to the "full P/S statement" right? Just want to be clear on that so contributors aren't confused down below (ex: `Email strategy@expensify.com and paste in the Proposal` - here they should include the entire P/S statement)

Just trying to clarify since in #whatsnext people often have a unique one-line "Proposal" statement, then they go on to the P/S statement

Reply:
Dang, it's a shame that we don't have the project process page of the manifesto public facing. I think that'd make this template significantly better. There's a lot of nuance that's hard to encapsulate in GH.

Reply:
>it's a shame that we don't have the project process page of the manifesto public facing. I think that'd make this template significantly better. There's a lot of nuance that's hard to encapsulate in GH.

I like this idea and I think we can be iterative here and improve things as we go. People will let us know what's ambiguous or requiring further clarification and we can choose to publish things as needed.